uamqp/constants.py

```diff
@@ -18,6 +19,8 @@
 OPERATION = b"operation"
 READ_OPERATION = b"READ"
 MGMT_TARGET = b"$management"
+EVENT_HUB_SCOPE = ['https://eventhubs.azure.net//.default']
+SERVICE_BUS_SCOPE = ['https://servicebus.azure.net//.default']
```

Comment:
Please remove these - we shouldn't keep references to Azure services in uAMQP. These should be passed through as values from EventHub or ServiceBus SDKs

Reply:
@annatisch Gotcha, just removed 😃

Reply:
I found Charles has updated the azure-identity, I need to do some modification to the JWTTokenAsync. I'll update you today when this pr is ready to merge.